### PR TITLE
Bump version to 0.26.0

### PR DESCRIPTION
**Pull request by EnricoMi**
Releasing v0.26.0.

Please pick your favourite features and note below to be highlighted in release title!
Example: https://github.com/horovod/horovod/releases/tag/v0.25.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,45 +8,70 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
-- Tensorflow: Added support for local variables for `BroadcastGlobalVariablesCallback`. ([3703](https://github.com/horovod/horovod/pull/3703))
+- Spark/Lightning: Spark/Lightning: add `reader_worker_count` and `reader_pool_type`. ([#3612](https://github.com/horovod/horovod/pull/3612))
+- Spark/Lightning: Added `transformation_edit_fields` and `transformation_removed_fields` param for `EstimatorParams`. ([#3651](https://github.com/horovod/horovod/pull/3651))
+- TensorFlow: Added doc string for `hvd.grouped_allreduce()`. ([#3594](https://github.com/horovod/horovod/pull/3594))
+- ROCm: Enable `alltoall`. ([#3654](https://github.com/horovod/horovod/pull/3654))
 
 ### Changed
 
 - Default Petastorm reader pool is changed from `process` to `thread` for lower memory usage. ([#3665](https://github.com/horovod/horovod/pull/3665))
+- Renamed `GPUReduceScatter` to `GPUReducescatter` for consistency. ([#3640](https://github.com/horovod/horovod/pull/3640))
```
Comment (maxhgerlach) [inline CHANGELOG.md:49]:
```suggestion
```
I'd leave this one out as it's a purely internal refactoring that's not visible to users.

**Review by maxhgerlach — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (EnricoMi) [reply CHANGELOG.md:49]: Is this not visible to user code?

Comment (maxhgerlach) [reply CHANGELOG.md:49]: Hmm, only to users who patch Horovod to add their own op implementations that derive from that internal class. https://github.com/horovod/horovod/blob/master/horovod/common/ops/gpu_operations.h#L254 

The rename was mostly to make it easier to find all related implementations by grepping the code base.

Comment (EnricoMi) [reply CHANGELOG.md:49]: Removing then.

**Review by romerojosh — APPROVED** [review]

**Pull request merged**